integration-tests/src/test/scala/org/constellation/testutils/HttpUtil.scala

```diff
@@ -0,0 +1,36 @@
+package org.constellation.testutils
+
+import cats.effect.{ContextShift, IO, Timer}
+import fs2.{Pipe, Stream}
+import org.http4s.client.blaze.BlazeClientBuilder
+import org.http4s.{EntityDecoder, Request}
+
+import scala.concurrent.ExecutionContext.global
+
+object HttpUtil {
+
+  implicit val cs: ContextShift[IO] = IO.contextShift(global)
+  implicit val timer: Timer[IO] = IO.timer(global)
```

Comment:
you can (or even should) pass the context shift and timer to the method directly and instantiate it once in the test